exercise/api/custom_serializers.py

```diff
@@ -49,10 +49,20 @@ def submission_url(submission_id):
                 'submission_id': submission_id
             }, request=request)
 
+        def submission_obj(submission_cached):
+            id_ = submission_cached['id']
+            return {
+                'id': id_,
+                'url': submission_url(id_),
+                'submission_time': submission_cached['date'],
+                'grade': submission_cached['points'],
+            }
+
         exercise_data = {
             'url': exercise_url(entry['id']),
             'best_submission': submission_url(entry['best_submission']),
             'submissions': [submission_url(s['id']) for s in entry['submissions']],
+            'submissions_with_points': [submission_obj(s) for s in entry['submissions']],
```

Comment:
Can you make this so, that we use the same urls for the other list too and don't do reverse map multiple times?

i.e.

```python
submissions = [submission_obj(s) for s in entry['submissions']]

       exercise_data = {
            'submissions': [s['url']) for s in submissions],
            'submissions_with_points': submissions,
```